go/flags/endtoend/vtctld.txt

```diff
@@ -1,5 +1,5 @@
 Usage of vtctld:
-      --action_timeout duration                                          time to wait for an action before resorting to force (default 2m0s)
+      --action_timeout duration                                          time to wait for an action before resorting to force (default 1m0s)
```

Comment:
Why is this changing? Can you also check the blame so that we can see whether it changed recently?
There's an action_timeout that defaults to 1h on the vtctlclient and vtctldclient binaries. How does this relate to those?

Reply:
This is a effect of https://github.com/vitessio/vitess/blob/8b8b135187bb8bf88409711e7d3948a5e953361c/go/vt/wrangler/wrangler.go#L39 and https://github.com/vitessio/vitess/blob/8b8b135187bb8bf88409711e7d3948a5e953361c/go/vt/vtctld/action_repository.go#L38

Apparently there are two action_timeouts 🤣. That action_timeout is a parameter for vtctldclient and vtctlclient. This one is taken by vtctld and applies only to `ApplyKeyspaceAction`, `ApplyShardAction` and `ApplyTabletAction`. This is the commit that introduced it - https://github.com/GuptaManan100/vitess/commit/facdb181da5c5adc680eca9cafc543086ad9e714